Community/kavSimulations/devices/kav_rudder.device.json

```diff
@@ -16,6 +16,11 @@
       "isI2C": false
     },
     "MessageTypes": [
+      {
+        "id": -1,
+        "label": "Clear display",
+        "description": "will clear the display"
+      },
       {
         "id": 0,
```

Comment:
I think we should review this. It might be better to work with SetFirstCharacter, and then use 0,1, or 2 for the Blank, Left, or Right.

Reply:
Must be checked in the code in detail